task/factory/default/src/main/java/io/knotx/fragments/task/factory/generic/GraphNodeOptions.java

```diff
@@ -35,6 +35,7 @@
 public class GraphNodeOptions {
 
   private NodeOptions node;
+  private String label;
```

Comment:
I think this label should be used in `GraphNodeExecutionLog`. Can you please verify if it is passed correctly?